conf/nginx/bedrock.conf.erb

```diff
@@ -0,0 +1,20 @@
+root /app/<%= ENV['DOCUMENT_ROOT'] %>;
+
+fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+index index.php index.html;
+
+location / {
+    try_files $uri $uri/ /index.php?$query_string;
+}
+location ~* ^.+.(jpg|jpeg|gif|css|png|js|ico|ttf|woff)$ {
+    access_log off;
+}
+location /. {
```

Comment:
Hmm I had it escaped in my comment - but it must have been removed by the parser.  Commit e57ea4b has the correctly escaped regex version.